models/staging/stg_dbt_source_freshness.sql

```diff
@@ -0,0 +1,44 @@
+with base as (
+
+    select * from {{ ref('stg_dbt_artifacts') }}
+
+),
+
+
+sources as (
+
+    select * from base
+    where artifact_type = 'sources.json'
+
+),
+
+
+fields as (
+
+    select
+        data:metadata:invocation_id::string as command_invocation_id,
```

Comment:
i dont think we need to reparse this in this model since the `command_invocation_id` is already parsed in `stg_dbt_artifacts`